termlink/__main__.py

```diff
@@ -129,6 +130,47 @@
 
 parser_gsea.set_defaults(execute=GSEACommand.execute)
 
+parser_snomedct = subparsers.add_parser(
+    "snomed-ct",
```

Comment:
Kebab casing or snake casing? Thoughts? 🤔

Reply:
I see other snaking casing for arguments ("--skip-synonyms"). So my vote is to keep it as snomed-ct but I can change it if you would like.